src/style.css

```diff
@@ -1,3 +1,77 @@
-body {
+:root {
+  --active: #00ACDC;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
it's not a good practice to reset styles at `*` selector. So, move `margin: 0; padding: 0;` to those tags you need